doc/source/user_guide/virtual_env_motorcad.rst

```diff
@@ -0,0 +1,61 @@
+.. _ref_virtual_env_motorcad:
+
+Using Python Virtual Environment in Motor-CAD
+===============================
+
+User have a flexibility to use python virtual environment with Motor-CAD. By default Motor-CAD uses python 
+comes with Motor-CAD installation. 
+
+.. figure:: ../images/defaults_python_path.PNG
+    :width: 500pt
+
+Creating a new local Python virtual environment
+*******************************
+If python is already installed we can check the python version by following command in a terminal
+(windows terminal or any terminal integrated with IDE such as VSCODE or PyCharm)
+
+.. code-block:: console
+
+    python --version
+
+Based on the above python version, a virtual environment can be created
+
+.. code-block:: console
+
+    python -m venv virtual_environment_folder_location
+
+Activate the newly created virtual environment
+
+.. code-block:: console
+
+     .\virtual_environment_folder_location\Scripts\activate.ps1
```

Comment:
This command is correct if the user is using powershell. If using the old 'cmd.exe' command line, then it's .\virtual_environment_folder_location\Scripts\activate.bat

Reply:
> This command is correct if the user is using powershell. If using the old 'cmd.exe' command line, then it's .\virtual_environment_folder_location\Scripts\activate.bat It's probably best to show both.

Agreed. You can use tabs to do this. Plenty of examples are available in the developer's guide.